src/core/search/indices.cc

```diff
@@ -2,6 +2,12 @@
 // See LICENSE for licensing terms.
 //
 
+// Wrong warning in a deeply inlined when boost::geometry::buffer is used.
+#ifndef __clang__
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wmaybe-uninitialized"
+#endif
+
```

Comment:
You can do it around the geo index and then do `diagnositc pop` or smth like this 😅